cellular_configs/v1alpha1/amfconfig_types.go

```diff
@@ -19,68 +19,58 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// AmfConfigStatus defines the observed state of AmfConfig
-type AmfConfigStatus struct {
+// AMFConfigStatus defines the observed state of AmfConfig
+type AMFConfigStatus struct {
 }
 
 // +kubebuilder:object:root=true
-type AmfConfig struct {
+type AMFConfig struct {
```

Comment:
Do we call this AMF as this is a subset of the AMF. Seems it is better called ServedGuami

Reply:
I just kept ServedGUAMI for the moment but later we can add more so it is better to call it AMF to allow expansion.